libs/community/langchain_community/document_loaders/confluence.py

```diff
@@ -152,6 +154,7 @@ def __init__(
         session: Optional[requests.Session] = None,
         oauth2: Optional[dict] = None,
         token: Optional[str] = None,
+        cookies: Optional[dict] = None,
```

Comment:
Can we make this parameter keyword-only (move below the `*`), in case users are passing in parameters positionally?

Reply:
Make sense, making `cookies` a keyword-only parameter is a backward compatible way, although semantically `cookies` are far away with authentication related parameters(`api_key`, `oauth2`, `token`)

Reply:
Done